### PR TITLE
Fix missing payer in kv snapshots and backing_store setting error in …

### PR DESCRIPTION
**Pull request by linhuang-blockone**
…kv_snapshot_test

## Change Description

Corrected backing_store error, added payer to kv_object_view, added code to convert payer representation between chainbase and rocksdb. 

## Change Type
**Select ONE**
- [ ] Documentation

- [ ] Stability bug fix

- [X] Other

- [ ] Other - special case

## Consensus Changes
- [ ] Consensus Changes

## API Changes
- [ ] API Changes

## Documentation Additions
- [ ] Documentation Additions

### CODE REVIEW

File: libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp [hunk 1]
```diff
@@ -7,21 +7,35 @@
 namespace eosio { namespace chain {
    static constexpr auto kv_payer_size = sizeof(account_name);
 
-   static uint32_t actual_value_size(const uint32_t raw_value_size) {
+   static uint32_t kv_rksdb_actual_value_size(const uint32_t raw_value_size) {
```
Comment (larryk85) [inline libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp:10]: Why were these functions renamed? The `kv_rksdb_` prefixes are a bit much and indicative that we should be namespacing these.  Was there clashing of function names?  Also, none of these functions are `inline`, meaning that they will cause duplicate symbols if we include in multiple compilation units.

**Review by larryk85 — CHANGES_REQUESTED** [review]

File: libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp (hunk 1 above)
Comment (linhuang-blockone) [reply libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp:10]: I thought the original names were too generic to be used outside kv_context_rocksdb. Will change to use name space and inline. Thanks.

Comment (brianjohnson5972) [reply libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp:10]: Can you revert them?  I renamed them in my branch.

Comment (brianjohnson5972) [reply libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp:10]: I renamed them and moved them, since I needed them as well.

**Review by brianjohnson5972 — CHANGES_REQUESTED** [review]
After you revert the name changes. I have a class to replace your new method, but I will do that when I merge, but would rather not have to fix all those merges if I can avoid it.

**Review by TimothyBanks — APPROVED** [review]

File: libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp (hunk 1 above)
Comment (linhuang-blockone) [reply libraries/chain/include/eosio/chain/backing_store/kv_context_rocksdb.hpp]: I am going to revert the naming changes, not introducing namespace for now so we can move forward quickly today. Will revisit this next week once everything is in.

**Review by larryk85 — APPROVED** [review]

**Review by brianjohnson5972 — APPROVED** [review]

**Pull request merged**